npm-shrinkwrap.json

```diff
@@ -8638,9 +9066,9 @@
       "integrity": "sha512-wPVv/y/QQ/Uiirj/vh3oP+1Ww+AWehmi1g5fFWGPF6IpCBCDVrhgHRMvrLfdYcwDh3QJbGXDW4JAuzxElLSqKA=="
     },
     "io.appium.settings": {
-      "version": "2.15.0",
-      "resolved": "https://registry.npmjs.org/io.appium.settings/-/io.appium.settings-2.15.0.tgz",
-      "integrity": "sha512-RT0Nn+N/FtuZoC1SUrDAs7LJ3ykZeCFkb2F3P5phPYQOIVTcicol5HoNlj5d7CuULFqZwY7oW/RS/lf1MRVNOA=="
+      "version": "3.0.0",
```

Comment:
3.0.0 bumped minSdkVersion. https://github.com/appium/io.appium.settings/pull/50/file
So, it's safe to update to 2.16.2 (not 3.0.0) for such patch release (2.16.2 has the location fix https://github.com/appium/io.appium.settings/commits/master )